pkg/cloud/ovirt/machine/actuator.go

```diff
@@ -295,6 +302,7 @@ func (actuator *OvirtActuator) reconcileNetwork(machine *machinev1.Machine, inst
 		klog.Errorf("failed to lookup the VM IP %s - skip setting addresses for this machine", err)
 		return err
 	} else {
+		klog.V(5).Infof("resolved IP address %v from DNS", ips)
 		for _, ip := range ips {
 			if ip.To4() != nil {
```

Comment:
do we intentionally ignore ipv6? why? what if it is ipv6-only?

Reply:
without a response, please do take a look if it supports IPv6 in general and if there would be a problem with that. It can happen that the only address you get is IPv6, and if there is no reason to limit it artificially it should be ok to remove this condition.

Reply:
okay lets take this into different task/PR ...